person.rb

```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initalize(_age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = int
+    @parent_permission = parent_permission
+  end
+
+  # rubocop:disable Naming/PredicateName
+
+  def can_use_services?
+    if is_of_age || @parent_permission
+      true
+    else
+      false
+    end
+  end
+
+  private
+
+  def is_of_age
+    is_of_age >= 18
```

Comment:
- [ ] Please go over this method again. Here you should be checking if ```@age >=18``` and not the method itself.